anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -1255,6 +1259,21 @@ def _sync_grype_groups(
                 )
             )
 
+        for source in source_counts:
+            if is_new:
+                updated_count = source.count
+            else:
+                updated_count = 0
+
+            results.append(
+                GroupSyncResult(
+                    group=source.group,
+                    updated_record_count=updated_count,
+                    total_time_seconds=total_time_seconds,
+                    status="success",
+                )
+            )
+
```

Comment:
If this block is executed in the happy path, it should be moved into try possibly before the db.commit()